documentation/site/pages/provers/quick-start.mdx

```diff
@@ -115,13 +115,13 @@ Once everything is up, you can send a test proof to Bento to see if everything w
 1. The Bento CLI allows for a simple test vector with a specified cycle count via the `-c` flag:
 
 ```sh [Terminal]
-RUST_LOG=info cargo run --bin bento_cli -- -c 32
+RUST_LOG=info cargo run --bin boundless-cli -- -c 32
```

Comment:
Should be bento CLI

Reply:
interesting, I was informed Lucian that it was renamed. That's my misunderstanding, and I should have verified.